Tools/machines/lxplus-cern/spack.yaml

```
@@ -0,0 +1,59 @@
+spack:
+  # variations of this environment can be controlled via environment variables during `spack env create`
+  definitions:
+  # CUDA
+  - cuda: []
+  - cuda: [cuda, blaspp +cuda, lapackpp ^blaspp +cuda]
+    when: env.get("SPACK_STACK_USE_CUDA", "") == "1"
+  - cuda: [cuda, blaspp, lapackpp]
+    when: env.get("SPACK_STACK_USE_CUDA", "") != "1"
```

If we want to make CUDA/Python on by default, we would do this:
```suggestion
    when: env.get("SPACK_STACK_USE_CUDA", "1") == "1"
  - cuda: [cuda, blaspp, lapackpp]
    when: env.get("SPACK_STACK_USE_CUDA", "1") != "1"
```

And then change the wording in the docs above accordingly.

---

Cool! I will also fix the documentation.